### PR TITLE
Navigation update

### PR DESCRIPTION
**Pull request by deronjohnson**
This is an update to the current mobile navigation. I kept the navgoco plugin intact, so it wouldn't affect any desktop functionality, although some of the style updates were applied to desktop as well. We're still doing some polishing on our end, but we wanted to get this in front of your team.

@jseldess @jess-edwards @kuanluo

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/1301)

### CODE REVIEW

**Comment by jseldess** [comment]
@deronjohnson, sorry for my delay! These updates look great to me, but I'd like @mjibson to review the changes as well. @mjibson, you up for that?

Otherwise, I'm testing this morning and will share any new findings with @kuanluo.

File: _includes/sidebar.html [hunk 1]
```diff
@@ -49,8 +49,37 @@
 
         {% include custom/conditions.html %}
 
-        <ul id="mysidebar" class="nav" style="display: none">
-            <li>
+        <ul id="mysidebar" class="nav {% if page.url == '/' or page.url == '/search.html' %}nav--home{% endif %}" style="display: none">
+            <div class="sidenav-arrow"><img src="images/sidenav-arrow.svg"></div>
```
Comment (jseldess) [inline _includes/sidebar.html]: Our CI platform (Teamcity) runs an HTML proofer against the Jekyll output. In addition to checking links, it checks that images have alt texts (for accessibility). It's detecting that the sidenav arrow image is missing one of those, so we need to either add an alt text field:

```
<img src="images/sidenav-arrow.svg" alt="Sidenav arrow">
```

or tell the HTML proofer to ignore the image:

```
<img src="images/sidenav-arrow.svg" data-proofer-ignore>
```

**Comment by jess-edwards** [comment]
@deronjohnson & @jseldess & @kuanluo 

The new nav is causing the page to wobble / bounce around from side to side as you scroll up and down. The page I am seeing this on is [Explore the Admin UI](https://www.cockroachlabs.com/docs/explore-the-admin-ui.html). The page does not wobble when you open the current site, so it leads me to think that it's being caused by the new nav. 

Device: iPhone 6s
Browser: Safari

**Comment by jess-edwards** [comment]
@deronjohnson The wobbling is much better! 

This gets an LGTM from me. 

@kuanluo @jseldess

**Comment by kuanluo** [comment]
LGTM as well! 👍

**Comment by mjibson** [comment]
I don't have time now to review this fully. I glanced at it and it looks fine. Fix the teamcity link errors and we will merge it.

**Comment by deronjohnson** [comment]
I'm not sure how to resolve this. It looks like all the errors from HTML proofer have to do with the internal hash links in sidebar.html, but I didn't touch any of the anchors in the update. How was this passing before?

**Comment by mjibson** [comment]
You're right, not your problem. We'll fix those elsewhere.

**Pull request merged**